llvm/test/CodeGen/AArch64/mlicm-csr-mask.mir

```diff
@@ -0,0 +1,49 @@
+# NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+# RUN: llc -mtriple=aarch64-unknown-linux-gnu -run-pass=greedy,machinelicm -verify-machineinstrs -debug -o - %s | FileCheck %s
+
+# FIXME: Running RA is needed otherwise it runs pre-RA LICM.
```

Comment:
but machinelicm should be the post-rA one? early-machinelicm is pre-RA

Reply:
For some reason it only triggers if I run RA first, otherwise it seems `MRI->isSSA()` returns true and that's what it uses to determine whether it's pre or post RA

Reply:
I think you are working around the MIR parser bug where it assumes SSA until it finds something obviously not SSA. The pass needs to clear the properties and I think we need an explicit SSA property